ConsoleApplicationGoBoard/ConsoleFunctions.cpp

```diff
@@ -4,7 +4,7 @@
 
 using namespace ::std;
 
-void ConsoleFuntions::PrintBoard(GoBoard &goBoard)
+void ConsoleFunctions::PrintBoard(GoBoard &goBoard)
```

Comment:
this could be a const function, because all you're doing is printing the board... this will lead to a lot of other changes in other functions that will need to become const